docs/support/plugin-support.md

```diff
@@ -68,3 +68,4 @@ For example, a plugin to provide a formatter which has itself been abandoned has
 | `hls-stan-plugin`                   | 3    | 8.6, 9.0, 9.2, 9.4       |
 | `hls-retrie-plugin`                 | 3    | 9.2, 9.4                 |
 | `hls-splice-plugin`                 | 3    | 9.4                      |
+| `hls-hectare-plugin`                | 3    | 8.10.7                   |
```

Comment:
Ah, maybe you got the semantics of this backwards: this is the list of _unsupported_ GHC versions.